### PR TITLE
First fix

### PR DESCRIPTION
**Pull request by ihernandez4466**
Made changes to the google map with markers of parking space locations.
Also, made changes to the listview of parking spots saved and pulled from Firebase.

### CODE REVIEW

File: parkcore_app/ios/Flutter/AppFrameworkInfo.plist [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>App</string>
 	<key>CFBundleIdentifier</key>
-	<string>io.parkcore.flutter.app</string>
+	<string>io.parkcore.flutter.app.app</string>
```
Comment (jcheon) [inline parkcore_app/ios/Flutter/AppFrameworkInfo.plist]: It should be:

<string>io.parkcore.flutter.app</string>

File: parkcore_app/ios/Podfile.lock [hunk 2]
```diff
@@ -170,42 +170,48 @@ PODS:
     - Flutter
```
Comment (jcheon) [inline parkcore_app/ios/Podfile.lock]: Dont include Podfile into commit.

File: parkcore_app/ios/Flutter/AppFrameworkInfo.plist (hunk 1 above)
Comment (jcheon) [inline parkcore_app/ios/Flutter/AppFrameworkInfo.plist]: It should be:

io.parkcore.flutter.app

File: parkcore_app/ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -8,17 +8,13 @@
```
Comment (jcheon) [inline parkcore_app/ios/Runner.xcodeproj/project.pbxproj]: Another file to not include to commit.

File: parkcore_app/pubspec.lock [hunk 4]
```diff
@@ -50,6 +50,48 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.4.2"
+  build_daemon:
```
Comment (jcheon) [inline parkcore_app/pubspec.lock]: Another file to not include to commit.

File: parkcore_app/pubspec.yaml [hunk 5]
```diff
@@ -44,6 +44,8 @@ dependencies:
   uuid: 2.0.4
 
 dev_dependencies:
+  pedantic: ^1.4.0
```
Comment (jcheon) [inline parkcore_app/pubspec.yaml]: should this be in dev_dependencies? or dependencies?

**Review by jcheon — CHANGES_REQUESTED** [review]
Thanks (:

File: parkcore_app/pubspec.yaml (hunk 5 above)
Comment (ihernandez4466) [reply parkcore_app/pubspec.yaml]: I took this off the instructions of putting google maps on flutter

**Pull request merged**